examples/flutter_firebase_login/lib/home/widgets/avatar.dart

```diff
@@ -9,10 +9,9 @@ class Avatar extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    final photo = this.photo;
     return CircleAvatar(
       radius: _avatarSize,
-      backgroundImage: photo != null ? NetworkImage(photo) : null,
+      backgroundImage: photo != null ? NetworkImage(photo!) : null,
```

Comment:
Personally I’d rather leave it as is because this is a sample and I would rather not encourage force unwrapping variables since that’s generally a dangerous thing to do.